src/server/node_services/nodes_monitor.js

```diff
@@ -1019,6 +1019,19 @@ class NodesMonitor extends EventEmitter {
             .then(() => this._update_status(item))
             .then(() => this._handle_issues(item))
             .then(() => this._update_nodes_store())
+            .finally(() => {
```

Comment:
What's the relation to _uninstall_deleting_node? Is that still relevant? (assuming no muli-PVs per pod)

Reply:
We do not wait for uninstall, it is not necessary

Reply:
then remove it